runtime/src/iree/hal/drivers/cuda/nccl_channel.c

```diff
@@ -122,7 +130,22 @@ static void iree_hal_cuda_nccl_channel_destroy(
   //  syms->ncclCommDestroy(channel->comm)
   // Should work the same (as we are doing a blocking teardown):
   //  syms->ncclCommDestroy(channel->comm)
```

Comment:
Is this comment stale?

Reply:
I left it as is for a todo, since the current code only supports the blocking mode now.